jobs/views.py

```diff
@@ -38,7 +38,11 @@ class JobCreate(CreateView):
     success_url = reverse_lazy('jobs:jobs')
 
     def form_valid(self, form):
-        company, created = Company.objects.get_or_create(name=form.cleaned_data['company_name'])
-        form.instance.company = company
-        form.instance.created = timezone.now()
+        job = form.save(commit=False)
```

Comment:
because this is a model form/view, we should save the instance first, with commit=False, then add in the company, then save it for real.